app/src/main/java/com/kickstarter/services/KSApolloClientV2.kt

```diff
@@ -1493,6 +1499,7 @@ class KSApolloClientV2(val service: ApolloClient, val gson: Gson) : ApolloClient
                 CreatePaymentIntentMutation.builder()
                     .projectId(encodeRelayId(createPaymentIntentInput.project))
                     .amount(createPaymentIntentInput.amount)
+                    .paymentIntentContext(StripeIntentContextTypes.POST_CAMPAIGN_CHECKOUT)
                     .build()
             ).enqueue(object : ApolloCall.Callback<CreatePaymentIntentMutation.Data>() {
```

Comment:
We should try to avoid using GraphQL defined models directly, take a look at @ycheng-kickstarter doc https://kickstarter.atlassian.net/wiki/spaces/NT/pages/2699100255/Event+Attribution+Engineering+Plan+AKA+How+To+Wire+Up+a+New+GraphQL+Endpoint+on+Android#3.-Create-a-CreateAttributionEventData-model-to-be-used-internally-which-matches-with-the-Apollo-generated-graphql-object-CreateAttributionEventInput